chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx

```diff
@@ -15,31 +14,34 @@ import useChatContextStore from "../../hooks/useChatContextStore";
 import useWindowDimensions from "../../hooks/useWindowDimensions";
 import { errorUILoadingPaneStyleProps } from "./common/errorUIStyleProps/errorUILoadingPaneStyleProps";
 
-export const LoadingPaneStateful = (props: ILoadingPaneProps) => {
+export const LoadingPaneStateful = (props: any) => {
```

Comment:
Is this not giving any warning while building?

Reply:
Added above eslint comment.